js/map_utils.js

```diff
@@ -187,7 +196,15 @@ function updateCategory(category, prevCategory) {
   updatePies(category, prevCategory);
   
   updateLegend(category);
-  
+  if(updateCategoryTimer){
+    clearTimeout(updateCategoryTimer);
+  }
+  updateCategoryTimer = setTimeout(function(){
```

Comment:
Possibly, it depends how much that delay turns out to be.  We just want try to collect events when the user is actually looking at a category, not if they just incidentally drag the cursor over something.

Reply:
right and I think that's exactly what we want for mouseovers - delay so that it only triggers when someone is actually hovering over that category wanting to inspect it and not accidentally passing over, but I could see still adding some extra delay to the ga event on top of the delay for the category switch